package.json

```diff
@@ -13,6 +13,11 @@
   "main": "dist/signature_pad.umd.js",
   "module": "dist/signature_pad.js",
   "types": "dist/types/signature_pad.d.ts",
+  "exports": {
+    "import": "./dist/signature_pad.js",
+    "require": "./dist/signature_pad.umd.js",
+    "default": "./dist/signature_pad.umd.js",
```

Comment:
Looks like it has a problem with the trailing comma I this line

Reply:
Fixed